### PR TITLE
Remove validation of removedPlugins in KubeSchedulerConfiguration

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind cleanup
/sig scheduling

#### What this PR does / why we need it:
We'll validate the removedPlugins in each KubeSchedulerConfiguration version, but in fact, there's no plugins to remove both in v1beta2 and v1beta3. Right now, we're ready to graduate KubeSchedulerConfiguration to v1, still no plan to remove plugin, they seem redundant then.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/apis/config/validation/validation_test.go [hunk 1]
```diff
@@ -278,10 +275,6 @@ func TestValidateKubeSchedulerConfigurationV1beta2(t *testing.T) {
 			expectedToFail: true,
 			config:         mismatchQueueSort,
 		},
-		"good-removed-plugins-2": {
-			expectedToFail: false,
```
[the remark this answers is not in the document]
Comment (kerthcet) [reply pkg/scheduler/apis/config/validation/validation_test.go:282]: We have no idea which plugin should exists or not, like out-of-tree plugin, so I think we can't cover this testcase. Actually, we only validate the in-tree plugin config in `validateKubeSchedulerProfile`.

[the remark this answers is not in the document]
Comment (alculquicondor) [reply pkg/scheduler/apis/config/validation/validation_test.go:282]: Nvm, I think that's not checked in validation_test.go

**Review by alculquicondor — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**